.github/workflows/release-mls-validation-service.yml

```diff
@@ -0,0 +1,45 @@
+name: Release MLS Validation Service
+
+on:
+  push:
+    tags:
+      - 'mls-validation-service-v*'
+
+  workflow_dispatch:
+
+jobs:
+  push_to_registry:
+    name: Push Docker Image to GitHub Packages
+    runs-on: warp-ubuntu-latest-x64-16x
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Log in to the container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/xmtp/mls-validation-service
+          tags: |
+            type=ref,event=tag
+            type=semver,pattern={{version}}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v6
+        id: push
+        with:
+          context: .
+          file: ./dev/validation_service/Dockerfile
+          push: true
```

Comment:
The one in xmtpd has `push: ${{ github.event_name != 'pull_request' }}`, is there a reason for the difference?

Reply:
this is copied from one place, and the other one from another place :)

In either case the event is not a pull request, so both will be true.